package.xml

```diff
@@ -17,19 +17,15 @@
   <url type="repository">https://github.com/uos/sick_tim</url>
 
   <buildtool_depend>catkin</buildtool_depend>
-
-  <build_depend>diagnostic_updater</build_depend>
-  <build_depend>dynamic_reconfigure</build_depend>
-  <build_depend>libusb-1.0-dev</build_depend>
-  <build_depend>roscpp</build_depend>
+  <build_export_depend>libusb-1.0-dev</build_export_depend>
   <build_depend>roslaunch</build_depend>
-  <build_depend>sensor_msgs</build_depend>
 
-  <run_depend>diagnostic_updater</run_depend>
-  <run_depend>dynamic_reconfigure</run_depend>
-  <run_depend>libusb-1.0-dev</run_depend>
-  <run_depend>robot_state_publisher</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>sensor_msgs</run_depend>
-  <run_depend>xacro</run_depend>
+  <depend>diagnostic_updater</depend>
+  <depend>dynamic_reconfigure</depend>
+  <depend>roscpp</depend>
+  <depend>sensor_msgs</depend>
+
+  <exec_depend>libusb-1.0-dev</exec_depend>
```

Comment:
I would propose to switch this to libusb-1.0

Reply:
of course, that was an oversight. Done.